pymilo/transporters/general_data_structure_transporter.py

```diff
@@ -85,7 +85,8 @@ def serialize(self, data, key, model_type):
         elif isinstance(data[key], np.int64):
             data[key] = {"value": int(data[key]), "np-type": "numpy.int64"}
 
-        # 4. list type which may containts unserializable type numpy.int32|int64
+        # 4. list type which may containts unserializable type
+        # numpy.int32|int64
```

Comment:
I believe it'd be better to remove all these comments. We already discussed them in the docstring.